notebooks/kaggle/store-sales-time-series-forecasting/Time_Series_Forecasting.ipynb

```diff
@@ -0,0 +1,1515 @@
scVjXOBc4r7T5xjQOB8Hfz5WLufZt8rqPHtD5I5obhJEARBEARBEIQixtCVj17+QqnJGtHVBl47bAeE93GhxcxMLWydFmE/rCBO29H5tgX9p/0ftT/MgtbZhf3CHofDDNnQ+lgZGg+E7gA1eCa0HP/9ng5YwirywP5WJyzH/N9Xpc+EAe3oPCC+hsgJS2fsmzQ9TjPad9r8t14GUOlhmMX+P+n271M1SY2p7D+rPVQY70cHsV94H4wlZE88P64DnUzKbOh1/n3b5LZvtPCeYy2eciXUIfu4eWG3WYX38riIe9Ird3Xd4hKUzHOh+Tc7YB/i/wSkZkB/B9DVbQvf7+uYFe177XCL3EPsB1LIsVm87ZNrF++ADZ1bzKH9IFXDZNADPe5Q2WSQqtMz75DyYy9s73UCcwzQifZmk4McOf11d8HSE+5z/QfaYf7IzUWUYbgrBg3cXn9v9aLLaUCWPujm4ThsHsZX3H286VCL2uu2W8Bt7aAcLTLmqmD5oA2Wd7TI+mHgMk3/8a7dLaztwcfl60Sp7pAiXDd7zUKUlGbC1dSAHUFXkcmKRxIkqz9zt9d6gwVOhj2jIqpPAYnEUNl2O3cWvDdOEd0gP2CHhbudWBHxt1XROKdS89s09HKX9Abh+jC0/yj1mRQ+w1Ejc9UaFFxmhqlp5MpdRbbweuBJyGcIgiAIgiAI4u+bMbT4qMFUNmG0vLGLJf3cBv0s2T9pQ+v6Zhzknj6RANo7CpB7shHl61phFc7tZpPQ1roq1L95DJ7Aaqc2BwXz3Wh9shqt7/fDHZChphbNR9Jh0AcenqCBdj5g62xD5/s22E8KsxL++y40lleMfN/dD+u2WlQ934FjHqEhl2eiYIkOlmdqYdrrFOqxo72uFo4pMm7SHOhDy4u12LTTDhf3XY8bzr2NaN4OGOfqhVvvtJi5QAXny1Wo3m6BzWaDZWc9KnY4/Q93EWG8pBf1L3aytgTOZ0LtMxboluTBELhFUW77RgnNVdnAvl3Y1S3U7XHBtq0azdZ4b44TJqVoRtXGwKQ0Fca7C6HeXYHyYX144DrcCdOGCrRYXfAOVxfBD6SQYbO42yfTLimnHejYWouGJgv63UK7bK0wbbFB+5CeeYxCLjEib7nIj1m9li0VqN2uRsE/ZzINK0eWnHzdarSvK4fp7YBOXbC/bUL9uhYcHPCG/ohxuRHZdzjR+HwjnDdnwRByq7ISm0ujmqpjcjHbhdiX9aE3TinXq4Budh5Ub7ejHTchI+gkWl0W3N0WWMW3jsvViVLdBaG+tQhrHgKaK+thDtxSLSsehZOM/uzXexfa/sDFuH5hsTFxe0ZCuj4FJBJD5dptkgY65he72tuH2+52mmF6dhdOKXDGhNsaZZyziHvFZToY5gCdDVVofNvK6rPCvLUaFWYv9EIRjrh95mIdFv6iBPqeWtRutfsXaeXawmtD8/LFWPxoM+y0AEkQBEEQBEEQcTGGFh/ZhHFpDQovNaN65RIsXsyS/ZIWDN5ZhGUGoUi8XGLAiudrkJ/SgQrh3EtWVqAjJR81jy8MeiJqKgyP1GD1PKCjfBWWcDIsK0PHuSysri2EPrjcA2uRf5EF9eVlKG0XJjPc8ZUNqLkvZeT7S1ahojMF+evLgza8T4F2QTlqlk6EtaZYqKcK1mklKMrlHvESndQbC1FZmgX3tlIs5767eAmKX7JB/Ugdim4NTPVToF9YiaI7U9D3cjXKysrQ+JEGBY8u4p9IKiY9twgFqi5ULRPOV2PFRHa+8gXaoIUIue0bHVLnLuN1Zq4S6l5cipYvclFUeINQIg7YpPS+fymA+p3gSWk+KhtKkHGkGaW8PrgHaTSjX1eCylXBC2qR/CAcOTaLv30y7cLaVb6+EBP3VWPVEqFdZR3AvNVYs0D0RFqZaO+txMbSDPS/JPgxq9fUrcKiZxLwB5lyDtfdFNDpcpQ29SOjlPm9+Am2TEczb8zm32XfaghfEJNt8whoc1Eism9VtwbL/j0Pk4Uiivm+Dlnc//O4RaARUq7WI5f9r5qbIV7Cka0TZboLJgW6+5bxV5HVPtXKXzEsLx6Fk5T+zPRe/AjXDi7GmWAJPFQoUXtGIlJ9skkshsqzmxa5pSXI+rxluO1LnrRAs6wIeVOEInJIuK1Rxrl7xL1CjeyfVaJA60JbXQWrrx4dg0asXp6FNKEER0I+c1kmCpZnwrWlCvX8Xr9ybZGGi7grmi9JC1wETBAEQRAEQRCEQr7D3XstvB8zeD0e/raxCy9ORYrcB6HIhbt9ij95jKdXn/PCM8QKjmPlQm7xEjHkgZeTU/g4Avu+vyKkRqsoUE8seSQJ1MF9XUoGAa6OcynS5x8wo3ppLdJrd7LJFvss6Ce27mW2bzQI2DCWbZIBs6/nnAx9RPQDMTJsllD75Nkl+X1sdPxBrpz+ckmqW67NpQj056+jX4iQq5Pk6U6Gb0uRlP7M6Z3FOKnvJ2LPiESpTzaJ9Rk5dhvuP0rsEUYS2ioa59x7q7GkBih5eTUyxU/DZv7gZb4Q1VZJHwMSswVBEARBEARBENEZk4uPxHlEvPhIEARBEKNI1MVHgiAIgiAIgiC+dYyh264JgiAIgiAIgiAIgiAIgvg2QYuPf++MV8OwtBDawENlCIIgCGIUSVEbULjUAPV44QBBEARBEARBEN9q6LZrgiAIgiAIgiAIgiAIgiBGBbrykSAIgiAIgiAIgiAIgiCIUYEWHwmCIAiCIAiCIAiCIAiCGBVo8ZEgCIIgCIIgCIIgCIIgiFGBFh8JgiAIgiAIgiAIgiAIghgVaPGRIAiCIAiCIAiCIAiCIIhRgRYfCYIgCIIgCIIgCIIgCIIYFWjxkSAIgiAIgiAIgiAIgiCIUYEWHwmCIAiCIAiCIAiCIAiCGBVo8ZEgCIIgCIIgCIIgCIIgiFGBFh8JgiAIgiAIgiAIgiAIghgVaPGRIAiCIAiCIAiCIAiCIIhRAPh/JJft3Qg2s1QAAAAASUVORK5CYII="
```

Comment:
I think you can copy-paste the text from kaggle istead of pasting a screenshot here. In markdown cells you can even make nice citations starting with `>` symbol, see this [cheat-sheet](https://www.markdownguide.org/cheat-sheet/)